### PR TITLE
convert auto-increment id to uuid

### PR DESCRIPTION
**Pull request by WilliamsOhworuka**
#### Pivotal tracker story
[#167711143](https://www.pivotaltracker.com/story/show/167711143)

#### What does this PR do?
   - Converts auto-incrementing id to UUIDs  
 
#### Summary of Task
   - install UUID from npm 
   - add UUIDs to controllers that create records

### CODE REVIEW

**Review by OvieMudi — APPROVED** [review]
LGTM

**Review by Nerocodes — APPROVED** [review]
LGTM

**Review by Rythae — APPROVED** [review]
LGTM

File: src/database/migrations/20190706140756-create-role.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+export const up = (queryInterface, Sequelize) => queryInterface.createTable('Roles', {
```
Comment (joel-ace) [inline src/database/migrations/20190706140756-create-role.js]: why are you adding this migration file?

**Pull request merged**